### PR TITLE
Upgrade tmux to version 2.0

### PR DESCRIPTION
**Pull request by jamezpolley**

### CODE REVIEW

File: Ports/tmux/Makefile [hunk 1]
```diff
@@ -5,10 +5,10 @@ include ../../Library/GNU.mk
 
 Title=		Terminal Multiplexer
 Name=		tmux
-Version=	1.9a
+Version=	2.0
 Revision=	0
 Site=		http://tmux.sourceforge.net/
-Source=		http://downloads.sourceforge.net/tmux/$(Name)-$(Version).tar.gz
```
Comment (ruda) [inline Ports/tmux/Makefile]: Are you having trouble with the previous Source URL? I prefer the original, it's shorter.

**Comment by ruda** [comment]
I will accept your contribution, since there isn't any guide line for the proper URL to sourceforge or gnu.org. Thanks!

**Comment by ruda** [comment]
My mistake, I merged with the wrong account, so I had to rebase the code of master. But your commit is preserved.

File: Ports/tmux/Makefile (hunk 1 above)
Comment (jamezpolley) [inline Ports/tmux/Makefile]: I was at the time, but checking again tonight the original seems to be working now.  I'll switch back, as I prefer the shorter version too.

Comment (jamezpolley) [inline Ports/tmux/Makefile]: But actually...

http://tmux.sourceforge.net now points at http://tmux.github.io which says the code is now at https://github.com/tmux/tmux so the correct download location is now https://github.com/tmux/tmux/releases/download/2.0/tmux-2.0.tar.gz

**Pull request merged**